.github/workflows/release.yml

```diff
@@ -3,16 +3,90 @@ name: Release
 
 on:
   pull_request:
-    types: closed
-
+    types: [ closed ]
 jobs:
   build:
     runs-on: ubuntu-latest
+    permissions: write-all
+    if: github.event.pull_request.merged
+    container:
+      image: pookmish/drupal8ci:latest
     steps:
-      - name: Tag
+      - name: Get Next Release
+        id: tag
         uses: K-Phoen/semver-release-action@master
         with:
+          release_strategy: none
           release_branch: develop
```

Comment:
Is this what limits this workflow to only merges on the `develop` branch?

Reply:
basically the `release_strategy` will give us the next version tag. We use that in next step because the release notes are better from the next step, but the next step doesn't act on the PR merge.